unetsocket/c/samples/npulses.c

```diff
@@ -0,0 +1,79 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+// Script to transmit a signal.
```

Comment:
We might want to explain in detail what `npulses` does.

Reply:
Added the explanation now.